addons/microcontroller/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Microcontroller
```

Comment:
Why not `ComputedProperties`? Also, I feel like `selections` should also be `computedProperties`. It better describes them, and would link it to the mixin, so it's clear they go with each other.